tests/core/editable/keystrokes/delbackspacequirks/manual/noselection.md

```diff
@@ -0,0 +1,14 @@
+@bender-tags: 4.7.0, tc, 13096
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar, wysiwygarea
+
+1. Open browser console.
```

Comment:
Actually I'm not able to create situation, in which there will be really no-selection in the editor.